packages/form-js-viewer/assets/form-js.css

```diff
@@ -235,4 +235,95 @@
 
 .fjs-container .fjs-form-field-text a {
   color: var(--color-blue-205-100-45);
-}
\ No newline at end of file
+}
+
+.fjs-container .fjs-taglist {
```

Comment:
That could need a format (indentation looks broken)

Reply:
I somehow cannot see this anywhere but on Github. Locally I have only spaces no tabs... Strange.